db/migrations/20220805145100_add_delete_on_cascade_to_fk_svc_inst_op_svc_instance_id.rb

```diff
@@ -0,0 +1,18 @@
+Sequel.migration do
+  # Add delete cascade to fk_svc_inst_op_svc_instance_id, without there won't be cleaned up all ocurrencies in
```

Comment:
I find this sentence hard to read. What about `Add DELETE CASCADE to foreign key fk_svc_inst_op_svc_instance_id to ensure that 'service instance operations' are deleted together with the referenced 'service instance'.`?

Reply:
ok, sounds good.